webpack/assets/javascripts/react_app/components/notifications/toast/Toast.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Icon from '../../common/Icon';
+import Alert from '../../common/Alert';
+
+const Toast = (props) => {
```

Comment:
I tried, but it was more than 4, that was both lint error and pretty long, there is probably a react way to describe the props - @gailsteiger ?